### PR TITLE
feat[module-ui]: Submitting UI sign in, sign up and sign out using HTML and CSS 

### PR DESCRIPTION
**Pull request by harshanooka**
Please merge the sign in, sign out and sign up for Movieflix using HTML and CSS

### CODE REVIEW

File: public/css/style.css [hunk 1]
```diff
@@ -0,0 +1,124 @@
...
+
+.tv-button{
+    padding-left: 2%;
+}
+
+.main-text-wrapper{
+    top:20%;
+    left: 11%;
+    position: absolute;
+    padding: 20px 20px;
```
Comment (kashyapmukkamala) [inline public/css/style.css]: Dont use hardcoded pixel values

**Comment by kashyapmukkamala** [comment]
👍

**Comment by kashyapmukkamala** [comment]
why is this closed?

**Comment by harshanooka** [comment]
Hi Kashyap,

I think it's open. There were two and I closed the old one so you should be
able to see the latest one. I even attached a screenshot, please have a
look at it.

**Pull request merged**